shakenfist_client/main.py

```diff
@@ -460,17 +461,17 @@ def network_show(ctx, network_uuid=None):
 
 @network.command(name='create',
                  help=('Create a network.\n\n'
+                       'NAME:             The name of the network\n'
```

Comment:
Isn't this a breaking UI change though?

Reply:
True but it is just a command line tool, not the API.

My argument is that:
* it's inconsistent with the other options
* there are few current users
* fix it early before changing the UI is painful

Being opinionated means we change things to be right. :-)

Reply:
Whilst giving arguments in the affirmative, I can be persuaded to remove the change. It's not a biggie for me.

Reply:
I guess we do already warn the user:

```
$ sf-client network create banana 192.168.1.42
ERROR: Malformed Request: cannot parse netblock: 'banana' does not appear to be an IPv4 or IPv6 network
```